### ISSUE TITLE
Async error : Polling is stopped

### ISSUE THREAD

**Issue by f48al** [eternnoir/pyTelegramBotAPI#1636]
Hello i'm using pyTelegramBotAPi version 4.6.0 and use python 3.10 with windows 10 as OS, i cannot run the bot using async because it always show error "Polling is stopped" any solution of this problem?
![IMG_20220718_113502_502](https://user-images.githubusercontent.com/31438208/179445895-6e80eb1b-2d4e-4349-aa83-4ee17dacebd8.jpg)

**Comment by coder2020official** [comment]
Just set non_stop=True, and next time your script will continue running.

**Comment by f48al** [comment]
> 

2022-07-18 05:43:37,165 (asyncio_helper.py:74 MainThread) ERROR - TeleBot: "Aiohttp ClientError: ClientConnectorCertificateError"
2022-07-18 05:43:37,165 (async_telebot.py:234 MainThread) ERROR - TeleBot: "Request timeout. Request: method=get url=getUpdates params=<aiohttp.formdata.FormData object at 0x0000018A384AB9D0> files=None request_timeout=20"

already try, but still can not receive the message

**Comment by coder2020official** [comment]
Try to set request_timeout to None.

**Comment by coder2020official** [comment]
The certificate error section looks like a bug. I'll investigate it.

**Comment by f48al** [comment]
> 

Yep i think its a bug, i tried everything that u tell to me but still can not receive the message. hope that u can fix it soon

**Comment by f48al** [comment]
> Try to install my github version and test.

already reinstall old version and install your github version and then run my script but it didn't worked and always show Certificate error
